cpp/vcpkg.json

```diff
@@ -56,11 +63,109 @@
     {
       "name": "openssl",
       "version-string": "1.1.1l"
-    }
+    },
+    {
+      "$comments": "All below are from the e82778a baseline. We have to use a newer baseline to pick up aws-c-sdkutils",
```

Comment:
Are you sure that this is the case? Can't we use the `overrides` feature? https://vcpkg.readthedocs.io/en/latest/users/manifests/#overrides Or, failing that, just upgrade the packages below in line with the baseline?

Reply:
Discussed and raised #238